tests/Games/NexusMods.Games.RedEngine.Tests/LibraryArchiveInstallerTests/ALibraryArchiveInstallerTests.cs

```diff
@@ -0,0 +1,194 @@
+using System.IO.Compression;
+using System.Text;
+using FluentAssertions;
+using Microsoft.Extensions.DependencyInjection;
+using NexusMods.Abstractions.GameLocators;
+using NexusMods.Abstractions.Jobs;
+using NexusMods.Abstractions.Library;
+using NexusMods.Abstractions.Library.Installers;
+using NexusMods.Abstractions.Library.Models;
+using NexusMods.Abstractions.Loadouts;
+using NexusMods.Extensions.Hashing;
+using NexusMods.Games.TestFramework;
+using NexusMods.Hashing.xxHash64;
+using NexusMods.MnemonicDB.Abstractions;
+using NexusMods.MnemonicDB.Abstractions.IndexSegments;
+using NexusMods.Paths;
+
+namespace NexusMods.Games.RedEngine.Tests.LibraryArchiveInstallerTests;
+
+public abstract class ALibraryArchiveInstallerTests : AGameTest<Cyberpunk2077.Cyberpunk2077Game>
```

Comment:
Do you want to move this to `NexusMods.Games.TestFramework`?

Reply:
I thought we'd keep it here until we need it in another test system, do you need it for SDV?

Reply:
moved